Dockerfile

```diff
@@ -4,14 +4,24 @@ ENV COMPOSER_IGNORE_PLATFORM_REQS=1
 WORKDIR /app
 COPY . /app
 RUN apk add --no-cache nodejs yarn
-RUN bin/build
+RUN --mount=type=cache,target=/app/vendor \
```

Comment:
This is not what we want to cache. `bin/build` script will install composer packages in `/app/build/manga-server/vendor`, therefore what we need to cache is `COMPOSER_CACHE_DIR` location. `composer` image doesn't set this env variable but instead set `COMPOSER_HOME` to `/tmp` directory.